build/Makefile

```diff
@@ -64,7 +64,7 @@ KIND_CONTAINER_NAME=$(KIND_PROFILE)-control-plane
 GS_TEST_IMAGE ?= us-docker.pkg.dev/agones-images/examples/simple-game-server:0.22
 
 # Enable all alpha feature gates. Keep in sync with `false` (alpha) entries in pkg/util/runtime/features.go:featureDefaults
-ALPHA_FEATURE_GATES ?= "PlayerAllocationFilter=true&PlayerTracking=true&CountsAndLists=true&FleetAllocationOverflow=true&Example=true"
+ALPHA_FEATURE_GATES ?= "CountsAndLists=true&DisableResyncOnSDKServer=true&Example=true&FleetAllocationOverflow=true&PlayerAllocationFilter=true&PlayerTracking=true"
```

Comment:
Can we please keep the original ordering, and append `DisableResyncOnSDKServer` on the end (or before `Example`) - makes things much easier to review and ensure nothing changed that it shouldn't.